src/msg/async/AsyncConnection.cc

```diff
@@ -327,6 +327,8 @@ void AsyncConnection::process()
   bool already_dispatch_writer = false;
   std::lock_guard<std::mutex> l(lock);
   last_active = ceph::coarse_mono_clock::now();
+
+  rcu_quiescent_state();
```

Comment:
I think we could call rcu_thread_offline() right before the call to driver->event_wait() , and then rcu_thread_online() when we return from it